content/push/device.textile

```diff
@@ -36,10 +36,21 @@ h3. Configure an iOS device
 
 * Go to the "Apple Developer Program.":https://developer.apple.com/programs/
 * Use the Apple Developer portal to create a *push notification* service certificate for your app.
-* Export the certificate as a *.p12* file.
-* In your Ably "dashboard":https://ably.com/accounts, navigate to the *notifications tab* under your app settings.
-* Go to *push notifications setup*, click *configure push*.
-* Add your *.p12* file.
+* Export the certificate as a *.p12 file*.
+* Next, you can either import the *.p12 cert* or create a *PEM file* and copy it into your dashboard:
```

Comment:
I think this be explicit that it's in your Ably dashboard given you were working in Apple's one beforehand.